src/server/dfly_main.cc

```diff
@@ -515,8 +515,10 @@ error_code GetCGroupPath(string* memory_path, string* cpu_path) {
   if (groups.size() == 1) {
     // for v2 we only read 0::<name>
     size_t pos = cgv.rfind(':');
-    if (pos == string::npos)
+    if (pos == string::npos) {
+      *memory_path = cgv;
```

Comment:
this is a bit hacky but I feel like it would save a bit of hassle in the future. Happy to hear some comments

(fyi: this is used in line 582)

Reply:
realistically this branch should never enter because that is simply the expected format of cgroups. this is assuming the function which reads is correct, but I think its a fair assumption :smile_cat: 

I guess originally I added that check as a sanity check. I will change this to `LOG(ERROR)` and exit, instead of returning.